File: src/Majority Element/Majority Element.py
class Solution:
	# @param num, a list of integers
	# @return an integer
	def majorityElement(self, num):
		cnt={}
		for c in num:
			if c in cnt:
				cnt[c] += 1
			else:
				cnt[c] = 1
			if cnt[c] > len(num)/2:
				return c

File: src/Majority Element/Majority Element.cpp
class Solution {
	public:
		int majorityElement(vector<int> &num) {
			map<int, int> cnt;
			int num_size = num.size();
			for(int i = 0 ; i < num_size ; i++)
			{   
				cnt[num[i]]++;
				if(cnt[num[i]] > (num_size>>1) ) return num[i];           
			}   
		}
};
